index.html

```diff
@@ -177,9 +177,35 @@ <h2 class="hability">Skills</h2>
         <li class="details">Terminal</li>
       </ul>
     </section>
+  </div>
 
+  <!--Contact form-->
+  <section class="aboutme_form">
+    <p class="formp">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+
+    <form class="about_form" action="https://formspree.io/f/xrgrjovw" method="POST">
+    
+      <input type="text" name="fullname" class="formname" maxlength="30" required placeholder="Full name">
+      <input type="email" class="formemail" required placeholder="Email address">
+      <textarea maxlength="500" class="textholder" placeholder="Enter text here" required></textarea>
+      
+      <section class="finalsection">
+        <button class="submit_but">Get in touch</button>
```

Comment:
- [ ] Follow the guidelines by implementing the [type attribute](https://www.w3schools.com/tags/att_input_type_submit.asp), setting it to `submit`.

Reply:
Thanks for the advice. Done.